packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-container.jsx

```diff
@@ -133,12 +142,14 @@ const ProofOfIdentityContainer = ({
                 getChangeableFields={getChangeableFields}
                 identity_last_attempt={identity_last_attempt}
                 idv={idv}
+                index={index}
```

Comment:
we don't have index in account->settings-> Proof of Identity. right?

what will be the value for index here?

Reply:
we have to pass in index from Verification modal to update the steps count. If not it won't get the value and add it to the steps

Reply:
yes. My concern was for account->settings-> Proof of Identity what will be the value of index since it is not a modal and we dont have steps index there?

Reply:
yeap, it will be undefined